src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+     action="/" 
+     method="GET"
+     data-qa="big"
+     class="form form--big"
+     > 
```

Comment:
1 tab = 2 spaces from open tag
use it in all cases
also, open teg & closed tag in the same vertical line
```suggestion
    <form 
      action="/" 
      method="GET"
      data-qa="big"
      class="form form--big"
    > 
```

Reply:
the closed tag should on the same vertical line with opened tag
```suggestion
    <form 
      action="/" 
      method="GET"
      data-qa="big"
      class="form form--big"
    > 
```